### PR TITLE
NO-1614 Correct the block of admins creating grants and warn on possible CSV line deletion.

### PR DESCRIPTION
**Pull request by scottynomad**
`hardhat --network localhost vesting create --diff --as-json --commit d2f08f31296262dec5b437009dcd307eef33f94f`
```Reading grants from github commit: d2f08f31296262dec5b437009dcd307eef33f94f
...

An unexpected error occurred:

ProviderError: Error: VM Exception while processing transaction: reverted with reason string 'lNORI: Recipient cannot be grant admin'
```

`hardhat --network localhost vesting create --diff --as-json`
```Reading grants from github commit: master

...

Found 9 grants that need updating
Total bpNORI to lock: 35079493.1786
⏰ Waiting for transaction (tx: 0x0d254cb52ae2a688167d84eeb2a11c17e56248e3723498a3827762765c89b3f9)
🎉 Created 9 grants (tx: 0x0d254cb52ae2a688167d84eeb2a11c17e56248e3723498a3827762765c89b3f9)
```

### CODE REVIEW

File: contracts/LockedNORI.sol [hunk 1]
```diff
@@ -530,7 +530,7 @@ contract LockedNORI is
       "lNORI: Recipient cannot be zero address"
     );
     require(
-      address(params.recipient) != _msgSender(),
+      !hasRole(TOKEN_GRANTER_ROLE, params.recipient),
```
Comment (jaycenhorton) [inline contracts/LockedNORI.sol:533]: Should probably prevent assigning this role to anyone with a grant as well

File: tasks/vesting.ts [hunk 2]
```diff
@@ -624,6 +624,14 @@ const GET_BLOCKCHAIN_SUBTASK = {
             } as any,
           };
     }, {} as ParsedGrant) as ParsedGrant;
+    const actualAmounts =  Object.values(rawBlockchainGrants).map(
+        ({grantAmount}) => grantAmount).reduce((acc, v) => acc.add(v));
+    if (!totalSupply.eq(actualAmounts)) {
```
[the remark this answers is not in the document]
Comment (jaycenhorton) [reply tasks/vesting.ts:629]: Really wanted to create a diff schema and use yup context to validate this kind of thing for consistency, but didn't have time

File: contracts/LockedNORI.sol (hunk 1 above)
Comment (scottynomad) [reply contracts/LockedNORI.sol:533]: Hmm, yeah that's a good idea.

File: tasks/vesting.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (scottynomad) [reply tasks/vesting.ts:629]: Oh you're right.  I have to subtract off all the claimed amounts. Good catch.

**Pull request merged**